src/Common/Modals/VisibleModal.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+export class VisibleModal extends React.Component<{
+    className: string
+    parentClassName?: string
+    noBackground?: boolean
+    close?: (e) => void
+    onEscape?: (e) => void
+}> {
+    modalRef = document.getElementById('visible-modal')
```

Comment:
Can we use refrence instead of `document.getElementById('visible-modal')` using js

Reply:
it is old code not changing this